.editorconfig

```diff
@@ -154,6 +154,9 @@ csharp_space_between_square_brackets = false
 # License header
 file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
 
+[src/coreclr/nativeaot/**/*.cs]
+dotnet_separate_import_directive_groups = true
```

Comment:
I think the repo preference is to put this in a local `.editorconfig`. So this would go to `src/coreclr/nativeao/.editorconfig`

Reply:
done